### PR TITLE
Adds spatialview API documentation #438

### PR DESCRIPTION
**Pull request by aj-he**
### brief description of changes

Adds spatial view API reference information

#### issues addressed

#438 

#### further comments

Relates to PR: https://github.com/archesproject/arches/pull/11101

---

This box **must** be checked
- [x] the PR branch was originally made from the base branch

This box **should** be checked
- [x] after these changes the docs build locally without error

This box **should only** be checked you intend to follow through on it (we can do it on our end too)
- [ ] I will `cherry-pick` all commits in this PR into other branches that should have them _after_ this PR is merged

### CODE REVIEW

**Comment by aj-he** [comment]
@ekansa - this is now ready for review with #435

File: docs/developing/reference/api.rst [hunk 1]
```diff
@@ -959,3 +959,220 @@ GeoJSON
...
+
+        curl -X DELETE http://localhost:8000/api/spatialviews/3d031564-3304-11ef-af57-0242ac150006
+
+        curl -H "Authorization: Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VySWQiOiJiMDhmODZhZi0zNWRhLTQ4ZjItOGZhYi1jZWYzOTA0NjYwYmQifQ.-xN_h82PHVTCMA9vdoHrcZxH-x5mb11y1537t3rGzcM" -X DELETE http://localhost:8000/api/spatialviews/3d031564-3304-11ef-af57-0242ac150006
+
+    **Example response**:
+
+    .. code-block:: http
+            
+        HTTP/1.0 204 OK
```
Comment (jacobtylerwalls) [inline docs/developing/reference/api.rst:1178]: Should this be no content instead of ok?

Comment (aj-he) [reply docs/developing/reference/api.rst]: Changed to `No Content`

**Review by ekansa — APPROVED** [review]
I'd love to actually play with this, but in terms of clear documentation this looks good!

**Pull request merged**